pyquil/api/_quantum_computer.py

```diff
@@ -158,7 +162,7 @@ def calibrate(self, experiment: Experiment) -> List[ExperimentResult]:
     def run_experiment(
         self,
         experiment: Experiment,
-        memory_map: Optional[Mapping[str, Sequence[Union[int, float]]]] = None,
+        memory_map: Optional[Dict[str, Union[Sequence[int], Sequence[float]]]] = None,
```

Comment:
Why the change from `Mapping` to `Dict`?

Reply:
No particular reason, just got clobbered by the new hint. Switched back to `Mapping` since it's more flexible.